syncopy/shared/computational_routine.py

```diff
@@ -291,17 +293,34 @@ def initialize(self, data, chan_per_worker=None, keeptrials=True):
             dtp_list.append(dtype)
             trials.append(trial)
 
+        # Determine trial stacking dimension and compute aggregate shape of output
+        if "time" in out_dimord:        # AnalogData + SpectralData + CrossSpectralData
```

Comment:
Might be that we have to come back here and add more variables?! Would it be maybe better to define a 'stackingDim' in the respective DataClass? It anyways knows it's own dimord of course, and hence can also inform the CR about it? It looks like the only use of the `out_dimord` is anyways exactly this no?

Reply:
Jupp, absolutley right. Godd point. We could add a `stackingDim` attribute to each of our classes - that might make things more transparent too. I would actually prefer this approach. Do you want me to go ahead and implement it?

Reply:
Yes please go ahead :smiley: !

Reply:
Done, please re-review